cmdeploy/src/cmdeploy/nginx/nginx.conf.j2

```diff
@@ -8,6 +10,21 @@ events {
 	# multi_accept on;
 }
 
+stream {
+        map $ssl_preread_alpn_protocols $proxy {
+            default 127.0.0.1:8443;
+            ~\bsmtp\b 127.0.0.1:465;
+            ~\bimap\b 127.0.0.1:993;
```

Comment:
can we use the /etc/service names of  "imaps" and "smtps" here? i'd prefer that.